modules/ROOT/pages/storage.adoc

```diff
@@ -354,6 +354,70 @@ storage:
       with_mount_unit: true
 ----
 
+== Adding Network Storage
+
+Fedora CoreOS systems can be configured to mount network file systems such as NFS and CIFS. This is best achieved by using Ignition to create systemd units. File systems can be mounted on boot by creating a standard mount unit. Alternately, a file system can be mounted when users access the mountpoint by creating an additional automount unit. Below are examples of each for an NFS file system.
```

Comment:
nit: we generally treat `filesystem` as one word.

Reply:
Changed to one word in all locations.